wallets/nwc/client.js

```diff
@@ -18,7 +18,7 @@ export async function validate ({ nwcUrl }, { logger }) {
   logger.ok(`connected to ${relayUrl}`)
 
   try {
-    return await new Promise((resolve, reject) => {
+    await new Promise((resolve, reject) => {
```

Comment:
did this because  `validate` now expects `undefined` or a new config returned. This didn't need to be returned anyway.